Integrations/CheckPhish/CheckPhish.py

```diff
@@ -0,0 +1,192 @@
+
+
+def is_valid_url(url):
+    try:
+        if 'http' not in url:
+            url = 'http://' + url
+        res = urlparse(url)
+        return all([res.scheme, res.netloc])
+    except ValueError:
+        return False
+
+
+def unite_dispositions():
+    for disp in GOOD_DISP:
+        DEFAULT_GOOD_DISP.add(disp)
+
+    for disp in SUSP_DISP:
+        DEFAULT_SUSP_DISP.add(disp)
+
+    for disp in BAD_DISP:
+        DEFAULT_BAD_DISP.add(disp)
+
+
+def get_dbot_score(disposition):
+    if disposition in DEFAULT_BAD_DISP:
+        return 3
+    if disposition in DEFAULT_SUSP_DISP:
+        return 2
+    if disposition in DEFAULT_GOOD_DISP:
+        return 1
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def test_module():
+    query = {'apiKey': API_KEY, 'urlInfo': {'url': 'https://www.google.com'}}
+    res = http_request('POST', BASE_URL, data=json.dumps(query))
+    if res and 'message' not in res:
+        return 'ok'
+    return res['message']
+
+
+def submit_to_checkphish(url):
+    if is_valid_url(url):
+        query = {'apiKey': API_KEY, 'urlInfo': {'url': url}, 'scanType': 'full'}
+        res = http_request('POST', BASE_URL, data=json.dumps(query))
+
+        if res:
+            return res['jobID']
+
+    else:
+        return_error(url + ' is not a valid url')
+        return False
+
+
+def get_status_checkphish(jobID):
+    query = {'apiKey': API_KEY, 'jobID': jobID}
+    res = http_request('POST', BASE_URL + STATUS_SUFFIX, data=json.dumps(query))
+
+    if res and res['status'] == DONE_STATUS:
+        return True
+    return False
+
+
+def get_result_checkphish(jobID):
+    query = {'apiKey': API_KEY, 'jobID': jobID}
+    res = http_request('POST', BASE_URL + STATUS_SUFFIX, data=json.dumps(query))
+
+    if res and 'errorMessage' not in res:
+        result = {'url': res['url'],
+                  'jobID': jobID,
+                  'status': res['status'],
+                  'disposition': res['disposition'],
+                  'brand': res['brand']}
+
+        url_dict = {'Data': result['url']}
+
+        if result['disposition'] != CLEAN_STATUS:
+            url_dict['Malicious'] = {'Vendor': 'CheckPhish',
+                                     'Description': 'Phishing countered to ' + result['brand']
+                                     }
+
+        dbot_score = {'Type': 'url',
+                      'Vendor': 'CheckPhish',
+                      'Indicator': result['url'],
+                      'Score': get_dbot_score(result['disposition'])}
+
+        context = {'CheckPhish.Url' + outputPaths['url']: result,
+                   outputPaths['url']: url_dict,
+                   'DBotScore': dbot_score}
+
+        human_readable = tableToMarkdown('CheckPhish reputation for: ' + result['url'],
+                                         result, ['url', 'disposition', 'brand', 'status', 'jobID'])
+
+        return_outputs(human_readable, context, res)
+
+    else:
+        return_error('Error getting job status')
```

Comment:
add the exact error

Reply:
Can't, this is the most informative error you can get from the API